bloom-instance/alb/inputs.tf

```diff
@@ -52,6 +52,16 @@ variable "internal" {
   description = "Whether this ALB is public or internal"
 }
 
+variable "enable_logging" {
+  type        = bool
+  description = "Whether to enable logging on this bucket"
```

Comment:
enable saving ALB logs to S3?

Reply:
I was asking if we could update the comment to be `Whether to enable enable saving ALB logs to S3`. 'this bucket' implies that the target resource is a bucket but it appears to be the logging flag on the ALB resource.

Reply:
Ah, good catch.  It really should say ALB instead of bucket.  Change incoming.

Reply:
The S3 part is implied since that's the only way to save ALB logs, so I kept it simple